openapi-cli/src/main/java/io/ballerina/openapi/cmd/OASContractGenerator.java

```diff
@@ -81,8 +83,13 @@ public List<OpenAPIConverterDiagnostic> getErrors() {
     public void generateOAS3DefinitionsAllService(Path servicePath, Path outPath, String serviceName,
                                                   Boolean needJson) {
         // Load project instance for single ballerina file
-        project = ProjectLoader.loadProject(servicePath);
-        Package packageName = project.currentPackage();
+        String customBallerinaHome = System.getProperty("ballerina.home");
+        Path ballerinaHome = customBallerinaHome == null ? null : Path.of(customBallerinaHome);
+        Environment environment = EnvironmentBuilder.getBuilder().setBallerinaHome(ballerinaHome).build();
```

Comment:
I suppose we don't have to manually set the ballerina home. shall we use the default environment?